wiremock-start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+nohup java -jar /home/runner/.m2/repository/org/wiremock/wiremock-standalone/3.9.2/wiremock-standalone-3.9.2.jar --port 18002 > wiremock.log 2>&1 &
+echo $! > wiremock.pid
```

Comment:
We write a file with the process id so we can stop the java process when the tests complete.